addons/spectator/functions/fnc_ui_updateIconsToDraw.sqf

```diff
@@ -139,9 +139,7 @@ private _camPos = AGLToASL positionCameraToWorld [0,0,0];
             SETVAR(_vehicle,GVAR(firedEH),_vehicle addEventHandler [ARR_2("Fired",{_this call FUNC(handleFired)})]);
         };
     };
-
-    nil // Speed loop
-} count ([] call FUNC(getTargetEntities));
+} forEach ([] call FUNC(getTargetEntities));
```

Comment:
I'm not sure what `if` returns when the then didn't run and there is no `else`. But I think there is a nil on the stack here meaning you can use `count`

Reply:
Just looked it up. `if` returns typeless nil when the condition didn't match and there was no else.